src/libs/ui/qxtglobalshortcut/qxtglobalshortcut_x11.cpp

```
@@ -59,15 +59,16 @@ bool QxtGlobalShortcutPrivate::isSupported()
 
 bool QxtGlobalShortcutPrivate::nativeEventFilter(const QByteArray &eventType,
-                                                 void *message, NativeEventFilterResult *result)
+    void *message, [[maybe_unused]] NativeEventFilterResult *result)
 {
-    Q_UNUSED(result)
-    if (eventType != "xcb_generic_event_t")
+    if (eventType != "xcb_generic_event_t") {
         return false;
+    }
 
     auto event = static_cast<xcb_generic_event_t *>(message);
```

Clang (14.0.6)

[warning] 'auto event' can be declared as 'auto *event'

(readability-qualified-auto)